action.yml

```
@@ -56,6 +56,10 @@ inputs:
     description: 'Whether to cache the Tailscale binaries (Linux/macOS) or installer (Windows)'
     required: false
     default: 'false'
+  targets:
+    description: 'Comma separated list of targets (IP addresses or machine name if MagicDNS is enabled on the tailnet) to ping for connectivity verification after `tailscale up` completes'
```

Maybe:

```suggestion
    description: 'Comma separated list of targets (Tailscale IP addresses or machine names if MagicDNS is enabled on the tailnet) to `tailscale ping` for connectivity verification after `tailscale up` completes'
```

to specify that we are attempting to reach these over `tailscale ping` specifically and that these targets need to be tailscale IP addresses / machine names specifically.